website/docs/reference/resource-properties/unit-tests.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "About unit tests property"
+sidebar_label: "unit tests"
+resource_types: [models]
+datatype: test
+---
+
```

Comment:
Hmm - do we think this is a property? It's more like a new resource type

Reply:
@graciegoheen I just cribbed from data tests which are listed as a property. Should I change just unit tests or both?

Reply:
I'm not actually sure! Just looking at all other pages in this section, they look like things you apply to the model/columns in your yml config:
```
models:
  - name: <model_name>
    config:
```

vs. unit tests which is a top level yml 
```
unit_tests:
...
```

What do you think? How do you define a "property"?

Reply:
Just checking back in about this! I think we decided to move this to a new "For unit tests" section (like how exposures is)

Reply:
> @matthewshaver is this updated re: our convo a couple of weeks ago? I _think_ we had decided to make this more like the "For exposures" property section.
> 
...
> 
> We may want to go ahead and rename the "For tests" section to "For data tests" as well. Lemme know what you think!
> 
> cc: @runleonarun in case you have thoughts here!

Hey @graciegoheen Sorry I missed the previous @mention. That makes sense to me. Are Unit tests like exposures in that they are "special properties?"  or are they [straight-up configs](https://docs.getdbt.com/reference/configs-and-properties#where-can-i-define-configs) that we can add to the dbt_project.yml? The example suggests configs because it's part of a project.yml file and top-level, but just want to double check. 

>Exposure properties are "special properties" in that you can't configure them in the dbt_project.yml file or using config() blocks. Refer to [Configs and properties](https://docs.getdbt.com/reference/configs-and-properties#which-properties-are-not-also-configs) for more info.

Reply:
I also added the "For unit Tests" folder to the Reference page which contains just the unit test file.

I also added a "Tests" folder to the /docs site under "Build your DAG" which contains unit test and data test files

Reply:
I like how this looks!